compiler-plugin/src/main/java/io/ballerina/stdlib/graphql/compiler/diagnostics/DiagnosticMessage.java

```diff
@@ -83,8 +83,9 @@ public enum DiagnosticMessage {
     ERROR_138("failed to add _service service to the subgraph service"),
     ERROR_139("failed to generate schema for type ''{0}''. Type alias for type ''{1}'' is not supported"),
     ERROR_140("invalid usage of @graphql:ID annotation. @graphql:ID annotation can only be used with string, "
-                      + "int, float, decimal and uuid:Uuid types");
-
+                      + "int, float, decimal and uuid:Uuid types"),
+    ERROR_201("invalid usage of `@deprecated` directive found in ''{0}''. Input object field(s) deprecation "
```

Comment:
Do we have to use back-ticks here?

Reply:
Removed in https://github.com/ballerina-platform/module-ballerina-graphql/pull/1518/commits/6c7e8043decd4f264b6fb5a4338067be8a18e6de